includes/class-orddd-lite-admin-delivery.php

```diff
@@ -223,6 +223,7 @@ public static function save_delivery_dates() {
 						( $previous_order_h_date == $_POST['h_deliverydate'] ) ) { // phpcs:ignore
 						$timestamp = Orddd_Lite_Common::orddd_lite_get_timestamp( $delivery_date, $date_format );
 						Orddd_Lite_Common::update_order_meta( $order_id, '_orddd_lite_timestamp', $timestamp, $order );
+						Orddd_Lite_Common::orddd_lite_changed_delivery( $previous_order_h_date, $timestamp );
```

Comment:
@nidhitatosaniya  Check the function orddd_lite_cancel_delivery() in class-orddd-lite-common.php. You can use that here.